# Spanish translations for PROJECT.
# Copyright (C) 2020 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
#
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2020-09-04 17:21+0500\n"
"PO-Revision-Date: 2020-09-04 17:23+0500\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: es\n"
"Language-Team: es <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.8.0\n"

#: app/api/v1/helpers/error_handlers.py:31
msgid "RESOURCE_NOT_FOUND"
msgstr ""

#: app/api/v1/helpers/error_handlers.py:86
msgid "method not allowed"
msgstr ""

#: app/api/v1/helpers/error_handlers.py:93
msgid "Registration Request not found."
msgstr ""

#: app/api/v1/helpers/error_handlers.py:94
msgid "De-Registration Request not found."
msgstr ""

#: app/api/v1/helpers/error_handlers.py:95
msgid "Report not found."
msgstr ""

#: app/api/v1/helpers/error_handlers.py:96
msgid "Report not allowed."
msgstr ""

#: app/api/v1/helpers/error_handlers.py:97
msgid "Document not found."
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:153
msgid "IMEIs per device count in file is not same as input."
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:155
#: app/api/v1/helpers/fileprocessor.py:175
msgid "Device count in file is not same as input"
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:157
#: app/api/v1/helpers/fileprocessor.py:177
msgid "Invalid IMEIs in the input file"
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:159
#: app/api/v1/helpers/fileprocessor.py:179
msgid "Duplicate IMEIs in the input file"
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:161
msgid "Some IMEIs are missing in the columns"
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:163
msgid "Rows limit is 10000000 for single request"
msgstr ""

#: app/api/v1/helpers/fileprocessor.py:165
#: app/api/v1/helpers/fileprocessor.py:181
msgid "Invalid IMEIs Format in input file"
msgstr ""

#: app/api/v1/helpers/utilities.py:291
msgid "size of file is greator than 26 MB which is not allowed"
msgstr ""

#: app/api/v1/helpers/validators.py:24
msgid "invalid imei"
msgstr ""

#: app/api/v1/helpers/validators.py:30
#, python-format
msgid "%(key)s value should be between 1 and 1000"
msgstr ""

#: app/api/v1/helpers/validators.py:32
#, python-format
msgid "%(key)s cannot start or ends with spaces"
msgstr ""

#: app/api/v1/helpers/validators.py:34
#, python-format
msgid "%(key)s cannot start or ends with tabs"
msgstr ""

#: app/api/v1/helpers/validators.py:36
#, python-format
msgid "%(key)s cannot end with line breaks"
msgstr ""

#: app/api/v1/models/deregdetails.py:221 app/api/v1/models/regdetails.py:184
msgid "The request is already closed"
msgstr ""

#: app/api/v1/resources/common.py:63
msgid "path cannot be empty"
msgstr ""

#: app/api/v1/resources/common.py:71
msgid "Unable to process the request"
msgstr ""

#: app/api/v1/resources/common.py:75
msgid "file not found or bad file path"
msgstr ""

#: app/api/v1/resources/deregdetails.py:64
#: app/api/v1/resources/deregdevice.py:51
#: app/api/v1/resources/deregdevice.py:100
#: app/api/v1/resources/deregdevice.py:157
#: app/api/v1/resources/regdetails.py:62
#: app/api/v1/resources/regdevicedetails.py:54
msgid "Failed to retrieve response, please try later"
msgstr ""

#: app/api/v1/resources/deregdetails.py:109
#: app/api/v1/resources/deregdetails.py:185
#: app/api/v1/resources/regdetails.py:108
#: app/api/v1/resources/regdetails.py:216
msgid "Registration request failed, check upload path or database connection"
msgstr ""

#: app/api/v1/resources/deregdetails.py:229
msgid "De-Registration request failed, check upload path or database connection"
msgstr ""

#: app/api/v1/resources/deregdocuments.py:56
#: app/api/v1/resources/regdocuments.py:62
msgid "Error retrieving results. Please try later."
msgstr ""

#: app/api/v1/resources/deregdocuments.py:103
#: app/api/v1/resources/regdocuments.py:110
msgid "request document addition failed, check for valid formats."
msgstr ""

#: app/api/v1/resources/deregdocuments.py:155
#: app/api/v1/resources/regdocuments.py:164
msgid "request document updation failed, please try again later."
msgstr ""

#: app/api/v1/resources/regdetails.py:173
msgid ""
"Registration update request failed, check upload path or database "
"connection"
msgstr ""

#: app/api/v1/resources/regdevicedetails.py:97
#: app/api/v1/resources/regdevicedetails.py:149
msgid "request device addition failed"
msgstr ""

#: app/api/v1/resources/restart_process.py:59
#: app/api/v1/resources/restart_process.py:105
msgid "This request cannot be processed"
msgstr ""

#: app/api/v1/resources/restart_process.py:69
#: app/api/v1/resources/restart_process.py:114
msgid "failed to restart process"
msgstr ""

#: app/api/v1/resources/reviewer.py:82 app/api/v1/resources/reviewer.py:105
#, python-format
msgid "incomplete request %(id)s can not be assigned/reviewed"
msgstr ""

#: app/api/v1/resources/reviewer.py:213 app/api/v1/resources/reviewer.py:243
msgid "request must be processed before reviewed."
msgstr ""

#: app/api/v1/resources/reviewer.py:755
#, python-format
msgid "unable to approve case %(id)s, duplicated imeis found"
msgstr ""

#: app/api/v1/resources/reviewer.py:894
msgid "Unable to approve, invalid imeis found"
msgstr ""

#: app/api/v1/schema/deregdetailsupdate.py:73
#: app/api/v1/schema/regdetailsupdate.py:88
msgid "The request status is already Closed"
msgstr ""

#: app/api/v1/schema/deregdetailsupdate.py:76
#: app/api/v1/schema/regdetailsupdate.py:91
#, python-format
msgid "The request status is %(status)s, which cannot be Closed"
msgstr ""

#: app/api/v1/schema/deregdetailsupdate.py:79
#: app/api/v1/schema/regdetailsupdate.py:94
msgid "The request is in Progress, which cannot be updated"
msgstr ""

#: app/api/v1/schema/deregdetailsupdate.py:82
#: app/api/v1/schema/deregdetailsupdate.py:85
#: app/api/v1/schema/deregdetailsupdate.py:88
#: app/api/v1/schema/devicedetailsupdate.py:58
#: app/api/v1/schema/regdetailsupdate.py:97
#: app/api/v1/schema/regdocumentsupdate.py:70
#, python-format
msgid "The request status is %(status)s, which cannot be updated"
msgstr ""

#: app/api/v1/schema/deregdevice.py:60 app/api/v1/schema/devicedetails.py:68
msgid "This step can only be performed for New Request"
msgstr ""

#: app/api/v1/schema/deregdocuments.py:59
#: app/api/v1/schema/deregdocumentsupdate.py:68
#: app/api/v1/schema/regdocuments.py:61
#: app/api/v1/schema/regdocumentsupdate.py:80
#, python-format
msgid "File format %(format)s is not allowed"
msgstr ""

#: app/api/v1/schema/deregdocuments.py:70
#: app/api/v1/schema/deregdocumentsupdate.py:79
#: app/api/v1/schema/regdocuments.py:72
#: app/api/v1/schema/regdocumentsupdate.py:91
msgid "File names should be unique"
msgstr ""

#: app/api/v1/schema/deregdocuments.py:100
#: app/api/v1/schema/regdocuments.py:102
msgid "This step can only be performed for request with Awaiting Document status"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:31
msgid "In Review"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:31
msgid "Approved"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:31
msgid "Rejected"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:31
msgid "Closed"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:32
msgid "New Request"
msgstr ""

#: app/api/v1/schema/devicedetailsupdate.py:32
msgid "Awaiting Documents"
msgstr ""

#: app/api/v1/schema/regdetails.py:78 app/api/v1/schema/regdetailsupdate.py:142
msgid "Duplicate IMEIs in request"
msgstr ""

